File: Unidad_1/01_TDA_Lenguaje_C/principal.c
#include "persona.h"
int main(int argc, char const *argv[]){
    
    persona *personas;
    personas=(persona*)malloc(sizeof(struct persona));
    personas->nombre=(char*)malloc(sizeof(char)*20);
    personas->apellido=(char*)malloc(sizeof(char)*30);
    personas->nombre="Kevin";
    personas->apellido="Celote Segundo";
    personas->edad=21;
    personas->genero='M';

    mostrarPersona(personas);
    comer(personas);
    caminar(personas);
    hablar(personas);
    dormir(personas);
    return 0;
}

File: Unidad_1/02_TDA_Lenguaje_JAVA/principal.java
public class principal{
    public static void main(String[] args) {
        persona personas = new persona("Kevin", "celote Segundo", 20,'M');
        personas.mostrarPersona();
        personas.comer();
        personas.caminar();
        personas.hablar();
        personas.dormir();
    }
}
